src/main/kotlin/solve/importer/ProjectParser.kt

```diff
@@ -45,25 +47,23 @@ object ProjectParser {
             landmark.projectLayer
         }.toSet())
         val diffString = diff.map { layer ->
-            layer.name + "_" + layer.kind
+            layer.name
         }
 
-        if (diff.isNotEmpty()) {
-            frame.isImageErrored = true
+        fun addNoSomeAlgorithmErrorMessage() {
+            val missingAlgorithms = diffString.toMutableList().toStringWithoutBrackets()
```

Comment:
why do we need mutable List?

Reply:
because `toStringWithoutBrackets`` takes mutable List. In most cases, it is  used and I decided not to write another function for a immutable List.